CMSgov/easi-app
# [EASI-3343] - update seed data

# EASI-3343

## Changes and Description

- Refactors System Intake Seed data to mainly use resolver code.
    - Helper functions for calling resolvers were split out into their own files.
    - Business Cases still rely on REST, so those functions were leveraged as-is.
    - I did have to move some of the system intake resolvers from `schema.resolvers.go` into  `/resolvers` so they would be available on the `resolvers` package.
- Adds seed data for as many permutations of the system intake process as possible.
    - I created static UUIDs for each new intake in case they need to be used in Cypress tests.
- Updates Legacy Seed Data to use the new methods.

## How to test this change

To see the data locally:

1. Start the app if necessary: `scripts/dev up`
2. Run: `scripts/dev db:seed`
3. Open [localhost:3000](http://localhost:3000)
4. Log in as `USR1`
5. View the generated intakes in both requester and admin views

To run the E2E tests:
1. Start the app if necessary: `scripts/dev up`
2. Run: `scripts/dev db:clean && scripts/dev db:seed && yarn cypress:run`

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
